lms/static/scripts/frontend_apps/components/ValidationMessage.js

```diff
@@ -33,11 +34,18 @@ export default function ValidationMessage({
   });
 
   return (
-    // FIXME-A11Y
-    // eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events
-    <div onClick={closeValidationError} className={errorClass}>
+    <button
+      // Add the aria-hidden="true" to inform axe tests to ignore it when 
```

Comment:
@robertknight Got to the button of this. I added the aria-label to make the axe happy w/o thinking too much about it because it told me it wanted a label, but it also said a few other things that spoke to the root of the issue.  When the validation message is closed, its not actually hidden because it needs to slide closed. So I needed to add 2 things to make this work correctly. 

Observing the docs here https://www.w3.org/TR/using-aria/#4thrule
Its seems reasonable to add `aria-hidden` directly yon the button. In this case, I don't think we need to cast to a string because I don't care about that attribute being set to "false" (that is an aside). But because its also a button, I needed to disable to the focus state when it was hidden so I added a dynamic tabindex too. axe tests now pass.